awx/ui/client/src/templates/job_templates/factories/md-5-setup.factory.js

```diff
@@ -10,8 +10,9 @@ export default
             master[check_field] = default_val;
 
             scope.genMD5 = function (fld) {
-                var now = new Date();
-                scope[fld] = md5.createHash('AnsibleWorks' + now.getTime());
+                scope[fld] = ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c =>
```

Comment:
This line of code is truly outstanding. A comment linking to the [original gist](https://gist.github.com/jed/982883) might be helpful for those curious about how this part works.